library/model/src/main/java/org/cru/godtools/model/Tool.kt

```diff
@@ -71,6 +72,9 @@ class Tool : Base() {
     @JsonApiAttribute(JSON_OVERVIEW_VIDEO)
     var overviewVideo: String? = null
 
+    @JsonApiAttribute(JSON_INITIAL_FAVORITES_PRIORITY)
+    var initialFavoritesPriority: Int? = Int.MAX_VALUE
```

Comment:
How is this applying an attribute to the tool exactly? Where does the Int value come from?

Reply:
the `@JsonApiAttribute` annotation is used by `JsonApiConverter` to serialize/deserialize json attributes. This property is only used within the initial-content module, and is read out of the `tools.json` file that gets bundled with the app at build time.